pkg/command/login.go

```diff
@@ -143,7 +142,7 @@ func prepareTanzuContextName(orgName, endpoint string, isStaging bool) string {
 		contextName += "-staging"
 	}
 
-	if endpoint != defaultTanzuEndpoint {
+	if endpoint != centralconfig.DefaultTanzuPlatformEndpoint {
 		// append just 8 chars of sha to the context name
 		contextName += fmt.Sprintf("-%s", hashString(endpoint)[:8])
```

Comment:
Currently, we are passing UCP endpoint here, but should the `context-name` include all other endpoint details so that if anyone of the 3 service endpoints gets updated, we create a new context?